src/app/core/data/collection-data.service.ts

```diff
@@ -57,4 +64,55 @@ export class CollectionDataService extends ComColDataService<Collection> {
       map((collections: RemoteData<PaginatedList<Collection>>) => collections.payload.totalElements > 0)
     );
   }
+
+  /**
+   * Fetches the endpoint used for mapping items to a collection
+   * @param collectionId   The id of the collection to map items to
+   */
+  getMappingItemsEndpoint(collectionId): Observable<string> {
+    return this.halService.getEndpoint(this.linkPath).pipe(
+      map((endpoint: string) => this.getIDHref(endpoint, collectionId)),
+      map((endpoint: string) => `${endpoint}/mappingItems`)
```

Comment:
I believe this endpoint path is incorrect. It's `/mappedItems`.  See https://github.com/DSpace/DSpace/pull/2282

Reply:
> I believe this endpoint path is incorrect. It's `/mappedItems`. See [DSpace/DSpace#2282](https://github.com/DSpace/DSpace/pull/2282)

Updated the endpoint's path, as well as all methods related to the endpoint, for more consistency.
While testing, I also discovered a bug in the collection-list on the item-collection-mapper component. Included a fix for that.